### PR TITLE
Georgian keyboard

### PR DESCRIPTION
**Pull request by arabuli**
Aded Georgian keyboard layout to Koreader.

---
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader/7225)

### CODE REVIEW

File: frontend/ui/language.lua [hunk 1]
```diff
@@ -19,6 +19,7 @@ local Language = {
         hu = "Magyar",
         nl_NL = "Nederlands",
         nb_NO = "Norsk",
+        ka = "ქართული",
```
Comment (Frenzie) [inline frontend/ui/language.lua:22]: Please put it above `nl_NL` for ease of maintenance.

File: frontend/ui/data/keyboardlayouts/keypopup/ka_popup.lua [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+    _r_ = {
+        "რ",
+        north = "ღ",
+    },
+    _S_ = {
+        "შ",
+    },
+    _s_ = {
+        "ს",
+        north = "შ",
```
Comment (Frenzie) [inline frontend/ui/data/keyboardlayouts/keypopup/ka_popup.lua:81]:
```suggestion
        north = "შ",
```

File: frontend/ui/data/keyboardlayouts/keypopup/ka_popup.lua [hunk 3]
```diff
@@ -0,0 +1,119 @@
+return {
+    _a_ = {
+        "ა",
+        north = "ჺ", -- georgian letter ain
+    },
+    _b_ = {
+        "ბ",
```
Comment (Frenzie) [inline frontend/ui/data/keyboardlayouts/keypopup/ka_popup.lua:7]: If there's only one character it's a bit weird to use a popup.

Basically either:
```lua
local _b_ = "ბ"
```

Or:

```lua
            { _B_,    "ბ",    ">",    prd,    "Ó",    "ó",    "†",    "‡", },
```

**Review by Frenzie — COMMENTED** [review]
Great, thanks! Just a few nitpicks.

**Comment by arabuli** [comment]
I changed some things and fixed the problems you mentioned. It should be good now.

**Comment by Frenzie** [comment]
Thanks!

**Comment by poire-z** [comment]
(Fixing luacheck's `frontend/ui/data/keyboardlayouts/ka_keyboard.lua:9:7: unused variable '_d_'` as part of #7226)

**Pull request merged**